x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -73,13 +77,13 @@ func (k Keeper) GetLastDataCommitment(ctx sdk.Context) (types.DataCommitment, er
 			return types.DataCommitment{}, err
 		}
 		if !found {
-			return types.DataCommitment{}, fmt.Errorf("couldn't find attestation with nonce %d", latestNonce-i)
+			return types.DataCommitment{}, sdkerrors.Wrapf(types.ErrAttestationNotFound, fmt.Sprintf("couldn't find attestation with nonce %d", latestNonce-i))
```

Comment:
```suggestion
			return types.DataCommitment{}, sdkerrors.Wrapf(types.ErrAttestationNotFound, fmt.Sprintf("nonce %d", latestNonce-i))
```

I think no need to repeat the message